tests/test_time_estimation.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+import unittest
+from tempfile import TemporaryFile
+
+from tests import VOSA_TESTFILES
+
+from partitura import load_musicxml
+from partitura.musicanalysis import estimate_time
+import partitura
+
+
+class TestTempoMeterBeats(unittest.TestCase):
+    """
+    Test tempo, meter numerator, and beat estimation.
+    """
+
+    score = load_musicxml(VOSA_TESTFILES[0])
+    tempometerbeats = estimate_time(score)
+
+    def testtempo(self):
+       
+        some_performance_notes = np.array([
```

Comment:
is it possible to save this array as a MIDI file on test files?

Reply:
sure this is possible, why?

Reply:
Maybe we could use it somewhere else too, but of course is not essential.